Simplified Installation: one line installation

NazarKY opened the pull request:
We should ask users minimum changes in the code. As for me for rails app will fine to have:

```
gem 'oauth2me'
```

will use default scopes and for each omniauth provider setup default api keys.

tmwh commented:
Currently to work with gem we need to add this:

``` ruby
OAuth2Me.setup_devise! unless Rails.env.production?
```

to devise initializer.

We need to add the railtie and create the initializer there.
http://api.rubyonrails.org/classes/Rails/Railtie.html

NazarKY committed:
2 simplify instalation:
simplify instalation in one line installation.
add checking for rails definding.

NazarKY commented:
@tmwh: pls review

tmwh commented on lib/oauth2-me.rb:
```diff
@@ -1,5 +1,6 @@
 require 'base64'
 require 'oauth2-me/omniauth_strategy_updater'
+require 'oauth2-me/railtie'
```
```
if defined?(Rails)
```

NazarKY commented:
@tmwh pls review
